packages/backend/src/config/queues.ts

```diff
@@ -1,5 +1,5 @@
 export const QUEUE_CONCURRENCY = {
-  POSTMAN_EMAIL: 3,
+  // POSTMAN_EMAIL: 3, // overridden in app config
```

Comment:
should we just remove this completely since its btr to just use based on our given rate limit